specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/common.json

```diff
@@ -99,15 +99,15 @@
         "Unauthorized",
         "Forbidden",
         "NotFound",
+		"TooManyRequests",
+		"InternalServerError",
```

Comment:
Make sure spacing is correct. Not sure for this repo, but some validation will make sure formatting is correct. The spacing looks off here, though it may just be GitHub's rendering.

Reply:
Oops, fixed.